src/main/webapp/app/eventMood.jsp

```diff
@@ -40,7 +40,7 @@
         <button type="button" onclick="back()">Go back!</button>
     </div>
     <div id="Submit">
-        <button type="button" onclick="submitVote()">Submit</button>
+        <button type="button" onclick="submitDailyVote()">Submit</button>
```

Comment:
Should probably call `submitEventVote()` - right now this page has the same function as `dailyMood.jsp`.

Reply:
This was planned to change later, but created an empty function for now and fixed eventMood.jsp called js function.